packages/common/src/services/resizer.service.ts

```diff
@@ -110,7 +115,13 @@ export class ResizerService {
 
     const containerNode = grid?.getContainerNode?.() as HTMLDivElement;
     this._gridDomElm = $(containerNode);
-    this._pageContainerElm = $(this._autoResizeOptions.container!);
+
+    if (typeof this._autoResizeOptions.container === 'string') {
+      this._pageContainerElm = $(this._autoResizeOptions.container);
+    } else {
+      this._pageContainerElm = $(this._autoResizeOptions.container!);
+    }
```

Comment:
I don't like this `if` here, but the typing on `$` doesn't support a parameter with `X | Y`.